date.pc.in

```diff
@@ -0,0 +1,10 @@
+prefix=@CMAKE_INSTALL_PREFIX@
+exec_prefix=@CMAKE_INSTALL_PREFIX@/bin
+libdir=@CMAKE_INSTALL_PREFIX@/lib@LIB_SUFFIX@
+includedir=@CMAKE_INSTALL_PREFIX@/include/date
```

Comment:
I think this should just be `@CMAKE_INSTALL_PREFIX@/include`  and then users are expected to `#include <date/date.h>`.  This follows https://github.com/HowardHinnant/date/blob/9a0ee2542848ab8625984fc8cdbfb9b5414c0082/CMakeLists.txt#L65.